modules/rostests/apitests/user32/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ list(APPEND SOURCE
     TrackPopupMenuEx.c
     VirtualKey.c
     WndProc.c
+    SW_FORCEMINIMIZE.c
```

Comment:
Would it make sense to move these tests inside the ShowWindow.c file?

Reply:
Moved.